### ISSUE TITLE
Windows network path support?

### ISSUE THREAD

**Issue by eitsupi** [rofinn/FilePathsBase.jl#166]
Hello. Sorry if this was not the proper place to submit this issue.

I tried to read a Parquet dataset on a network drive using Parquet2.jl, but it did not work.

```julia
using FilePathsBase
using Parquet2: Dataset

ds = Dataset("//path/to/file")
```

Perhaps paths on Windows network drives are not supported?

**Comment by rofinn** [comment]
Yeah, there's an open MR to support UNC paths. I don't use windows regularly, so you might want to comment on this MR https://github.com/rofinn/FilePathsBase.jl/pull/12
